File: Aprendendo-C++/Exercícios/crescente.cpp
#include <bits/stdc++.h>

using namespace std;

int main(){
    int m, n;

    cout << "Digite dois numeros:" << endl;
    cin >> n;
    cin >> m;

    while( m != n){
    
    if( n < m) {
        cout << "Crescente\n";
    } else if( m < n){
        cout << "Descrescente\n";
    }
    cout << "Digite outro dois numeros:" << endl;
    cin >> n;
    cin >> m;
    }

    return 0;
}

File: Aprendendo-Java/Exercícios/retangulo/src/exercícios/crescente.java
package exercícios;
import java.util.Scanner;

public class crescente{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        
        System.out.println("Digite dois numeros: ");
        int num = sc.nextInt();
        int num2 = sc.nextInt();
        
        while(num != num2){
        
        if(num > num2){
            System.out.println("Descrescente");  
            
        }else{
            System.out.println("Cescente");
        }
            
        System.out.println("Digite dois numeros: ");
        num = sc.nextInt();
        num2 = sc.nextInt();
        
        
        
   }
  }
}